best-practices/native/react-native/deployment/ios/README.md

```diff
@@ -1,79 +1,26 @@
-
-## Android Deployment
-
-
-
-### Setting up Google Play
-
-###### In the [Google Play Console](https://play.google.com/apps/publish/ "Google Play Console")
-- Select 'Create Application'
-- Name Your Application
-
-Before you can upload a build to Google Play, You'll need to set up the minimum allowable information about your application. The list of requirements may change, but currently it is:
-
-1. Add a hi-res icon
-	- must be 512x512px 32-bit PNG. This can contain an alpha channel, but since alpha channels (aka transparency layer) are not allowed in iOS store apps, it's best to create an icon with no alpha channel to prevent different icons on android/iOS
-
-2. Add a feature graphic
-	- 1024x500 JPG or PNG (no alpha channel allowed)
-3. Add at least 2 screenshots
-	- No alpha channel allowed
-
-4. Select an application category
-
-5. Add short and long descriptions of your application
-
-6. Provide a privacy policy URL
-
-7. Provide a content rating
-	- Google Play offers a series of questions that will automatically provide a content rating for you
-
-### Creating an APK
-
-#### What is an APK?
-The APK is the file that represents your application. It is uploaded to the Google Play Store and is eventually downloaded on the user's phone, letting them run your application. Emulators and dev-mode phones can run unsigned APKs, but you'll need to create a signed APK to distribute your app to users.
-
-#### What is a signed APK?
-A signed APK is an APK that has an attached keystore file. The keystore file contains some additional information about the app and the app uploader. If you're putting the app on the Play Store for the first time, you'll need to create a keystore file. If you're updating an existing app, you'll need the existing keystore before you can update the app on the Google Play Store. **IMPORTANT NOTE:**  Make sure you backup your Keystore after creating it. Once an app is published to the Play Store, it will be impossible to update it if you lose the keystore file or password.
-
-#### Steps to Create a signed APK
-###### In Android Studio:
-- Build -> Generate Signed APK
-- Android Studio will ask for the path to your keystore file and its password. If you don't have one, create one here.
-- Set 'Build Type' to 'release' (make note of the APK Destination Folder)
-- Make sure 'V1' and 'V2' Signature Versions are both checked.
-- Click 'finish'
-- After it's done, the APK will be available in the APK Destination Folder
+1. Open the xcode workspace project.
+2. In upper left-hand of screen, click on `Generic iOS Device`. Select a device you'd like to test on.
+3. Build the app via the play button. Be sure the iOS device is set to a physical apple device
+<img width="399" alt="Screen Shot 2021-07-30 at 2 23 02 PM" src="https://user-images.githubusercontent.com/25145605/127720045-664ec7b6-95e1-43b5-af55-9938b5c10c5d.png">
```

Comment:
Very helpful images. Nice job.

Reply:
I believe most of these were added by @gabymoreno9. We pushed for more docs within the Sundial repo and it did help out later on. I just pulled them all in here and organized a bit.